File: conf/php/structure/samples/TestApi.php
<?php

include_once( dirname(__FILE__).'/../APIClient.php');
include_once( dirname(__FILE__).'/../AntAPI.php');
include_once( dirname(__FILE__).'/../StorageAPI.php');

    $privateKey = "<PRIVATE_KEY>";
	$userId = "<CLIENT_ID>";
	$fileId = "<FILE_ID>";
	$apiClient = new APIClient($privateKey, "https://dev-api.groupdocs.com/v2.0");

	$api = new AntAPI($apiClient);

    // $postData = new AnnotationInfo();
    // $postData->Type = "0";
    // $box = new Rectangle();
    // $box->Height = 200;
    // $box->Width = 200;
    // $box->X = 200;
    // $box->Y = 200;
    // $postData->Box = $box;
    // $reply = new AnnotationReplyInfo();
    // $reply->Message = "sent from php client";
    // $postData->Replies = array($reply);
    // $response = $api->CreateAnnotation($userId, $fileId, $postData);
	// print_r($response);

	// $response = $api->ListAnnotations($userId, $fileId);

	// test file upload
	$api = new StorageAPI($apiClient);
	$api->Upload($userId, "test.docx", "uploaded", "file://".dirname(__FILE__)."/test.docx");

?>

File: conf/python/structure/samples/TestApi.py

import sys
import os
import hmac

from AntAPI import AntAPI
from APIClient import APIClient

sys.path.append(os.path.dirname(os.path.abspath(__file__)) + '/../')
import model


def listAnnotations():
    response = AntAPI(apiClient).ListAnnotations(userId, fileId)
    print(response.Status)

def createAnnotation():
    box = Rectangle()
    box.X = 100
    box.Y = 100
    box.Width = 100
    box.Height = 100
    reply = AnnotationReplyInfo()
    reply.Message = "Test message from python client"
    postData = AnnotationInfo()
    postData.Type = "0"
    postData.Box = box
    postData.Replies = [reply]

    response = AntAPI(apiClient).CreateAnnotation(userId, fileId, postData)
    print(response.Status)

def uploadFile():
    postData = "file://" + os.path.dirname(os.path.abspath(__file__)) + "/test.docx"
    response = StorageAPI(apiClient).Upload(userId, "python/test.docx", "uploaded from python client library", postData)
    print(response)


if __name__ == '__main__':
    privateKey = "<PRIVATE_KEY>"
    userId = "<CLIENT_ID>"
    fileId = "<FILE_ID>"
    apiClient = APIClient(privateKey, "https://dev-api.groupdocs.com/v2.0")

#    createAnnotation()
#    listAnnotations()
    uploadFile()

